docs/3.0/dashboards.md

```diff
@@ -55,77 +55,6 @@ The `id` field has to be unique. The `name` is what the user sees in big letters
 
 Using the ' grid_cols ' parameter, you may organize the cards in a grid with `3`, `4`, `5`, or `6` columns using the `grid_cols` parameter. The default is `3`.
 
```

Comment:
Whenever we take out a big chunk of documentation to it's own page, let's leave the heading there (for SEO and findability purposes) with a link to the new section.

Reply:
Added a reference in [this commit](https://github.com/avo-hq/avodocs/pull/110/commits/deb0295885976c8636d55e9a04d5ba813c9696a0)